src/darwin/Framework/CHIP/MTRDevice.h

```diff
@@ -29,6 +29,20 @@ typedef NS_ENUM(NSUInteger, MTRDeviceState) {
     MTRDeviceStateUnreachable = 2
 };
 
+/**
+ * This bitmask is used to specify the type of logs requested from this device.
+ *
+ * The logs types are : End User Support, Network Diag and Crash logs.
+ */
+typedef NS_OPTIONS(NSUInteger, MTRDiagnosticLogTypes) {
+    MTRDiagnosticLogTypeEndUserSupport = 0, // End user support logs are requested
+    MTRDiagnosticLogTypeNetworkDiag = 1 << 0, // Network Diag logs are requested
```

Comment:
```suggestion
    MTRDiagnosticLogTypeNetworkDiagnostics = 1 << 1, // Network Diagnostics logs are requested
```

Reply:
i did have it as Diagnostics before but then the word Diagnostic was twice in the word. makes sense though